daemon/monitor.go

```diff
@@ -324,24 +345,29 @@ func cleanupPortTable(vrsConnection vrsSdk.VRSConnection, entityPortList []strin
 	stalePorts := computeStalePortsEntitiesDiff(vrsPortsList, entityPortList)
 	deleteStalePortsList = getStalePortEntriesForDeletion(stalePorts)
 	for _, stalePort := range deleteStalePortsList {
-		log.Infof("Removing stale port %s", stalePort)
-		err = vrsConnection.DestroyPort(stalePort)
-		if err != nil {
-			log.Warnf("Unable to delete port from Nuage Port table: %v", err)
-		}
+		if strings.Contains(stalePort, "nu") {
```

Comment:
this matches all port names that has "nu" at any position in the string. can it be more specific and strict?